server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectionAlgorithm.java

```diff
@@ -200,17 +166,61 @@ private void confirmDeletes(GarbageCollectionEnvironment gce,
         String dir = reference.substring(0, reference.lastIndexOf('/'));
         if (candidateMap.remove(dir) != null)
           log.debug("Candidate was still in use: {}", reference);
+      }
+    }
+  }
+
+  private long removeBlipCandidates(GarbageCollectionEnvironment gce,
+      SortedMap<String,String> candidateMap) throws TableNotFoundException {
+    boolean checkForBulkProcessingFiles = false;
+    long blipCount = 0;
+    Iterator<String> relativePaths = candidateMap.keySet().iterator();
+
+    while (!checkForBulkProcessingFiles && relativePaths.hasNext())
+      checkForBulkProcessingFiles |=
+          relativePaths.next().toLowerCase(Locale.ENGLISH).contains(Constants.BULK_PREFIX);
```

Comment:
Could possibly shorten this with stream anyMatch()

Reply:
Yeah, that cleans it up nicely.